composer.json

```diff
@@ -45,6 +45,8 @@
         "jms/di-extra-bundle": "~1.7",
         "sensio/generator-bundle": "~2.3|~3.0",
         "symfony/yaml": "~2.3|~3.0",
+        "symfony/security-acl": "~2.4|~3.0",
+        "phpunit/phpunit": "~4.3",
```

Comment:
> Its very usefull to provide a php.ini precision for unittests, so that everyone has the same environment

Ok, so a good commit message would be : 

```
explicitely rely on a specific precision

This precision is implicitely relied on in some tests.
```